semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/cfgdominator/CfgDominator.scala

```diff
@@ -1,6 +1,6 @@
 package io.shiftleft.semanticcpg.passes.cfgdominator
 
-class CfgDominator[Node](adapter: CfgAdapter[Node]) {
+class CfgDominator[N](adapter: CfgAdapter[N]) {
```

Comment:
Please leave the speaking type parameter names in place. If `Node` collides with rename it to `NodeType`.